modules/openxAnalyticsAdapter.js

```diff
@@ -634,7 +667,405 @@ adapterManager.registerAnalyticsAdapter({
   code: 'openx'
 });
 
+const {
+  EVENTS: { AUCTION_INIT, BID_REQUESTED, BID_RESPONSE, BID_TIMEOUT, BID_WON }
+} = CONSTANTS;
+
+const ENDPOINT = 'https://prebid.openx.net/ox/analytics';
```

Comment:
@kenan-gillet this is the endpoint we're trying to hit, i dont believe we have this setup right?